src/main/java/com/minecolonies/api/compatibility/Compatibility.java

```diff
@@ -120,10 +120,10 @@ public static boolean isTinkersWeapon(@NotNull final ItemStack stack)
     /**
      * Check if a certain item stack is a tinkers tool of the given tool type.
      * @param stack the stack to check for.
-     * @param toolType the tool type.
+     * @param toolId the tool type ID.
      * @return true if so.
      */
-    public static boolean isTinkersTool(@Nullable final ItemStack stack, IToolType toolType) { return tinkersCompat.isTinkersTool(stack, toolType); }
+    public static boolean isTinkersTool(@Nullable final ItemStack stack, final String toolId) { return tinkersCompat.isTinkersTool(stack, toolId); }
```

Comment:
Can't we still hand around the actual type or registry entry? Not a huge fan of handing around the strings

Reply:
Done. I was worried about capturing the ToolTypeEntry that wasn't assigned yet but I tested and it works fine,